### PR TITLE
Upgrade GitHub Actions and PyPy 3.10 and Django 5.0

### PR DESCRIPTION
**Pull request by cclauss**
Upgrading our project from Django v4.2 to v5.0 ___downgraded___ our versions of django-celery-beat and django-timezone-field!

`Pipfile.lock`:
```diff
        "django-celery-beat": {
            [ ... ]
-           "version": "==2.5.0"
+           "version": "==2.1.0"
        },
```
% `python3.12 -m venv .venv`
% `source .venv/bin/activate`
% `pip install --upgrade pip`
% `pip install django==5.0 django-celery-beat==2.5.0` --> **_Failure_**
% `pip install django==5.0 django-celery-beat` --> Success but with
* `django-celery-beat` downgraded `2.5.0` --> `v2.1.0` and
* `django-timezone-field` downgraded from `v6.1.0`--> `v4.2.3`

Please release a new version to PyPI.
* #680
* #698

### CODE REVIEW

**Comment by cclauss** [comment]
@auvipy Can you please approve the workflow runs so I can see if these tests pass?

**Comment by Koff** [comment]
Could we get a status update on this? I want to upgrade to Django 5, but I can't because of the incompatibility with `django-celery-beat` latest.

**Comment by ddoroshev** [comment]
image

File: tox.ini [hunk 1]
```diff
@@ -72,7 +74,7 @@ commands =
 basepython = python3.9
 usedevelop = true
 commands =
-    pip install -U https://github.com/celery/celery/zipball/master#egg=celery
-    pip install -U https://github.com/celery/kombu/zipball/master#egg=kombu
+    pip install --upgrade https://github.com/celery/celery/zipball/master#egg=celery
```
Comment (auvipy) [inline tox.ini:77]:
```suggestion
    pip install --upgrade https://github.com/celery/celery/zipball/main#egg=celery
```

File: tox.ini [hunk 2]
```diff
@@ -72,7 +74,7 @@ commands =
...
+    pip install --upgrade https://github.com/celery/kombu/zipball/master#egg=kombu
```
Comment (auvipy) [inline tox.ini:78]:
```suggestion
    pip install --upgrade https://github.com/celery/kombu/zipball/main#egg=kombu
```

**Review by auvipy — CHANGES_REQUESTED** [review]
can you please check the pypy failure?

**Review by auvipy — APPROVED** [review]
good decision and great job

**Pull request merged**